etc/containerpilot.json5

```diff
@@ -0,0 +1,34 @@
+{
+  consul: "127.0.0.1:8500",
+  jobs: [
+    {
+      name: "preStart",
+      exec: ["/usr/local/bin/consul-manage", "preStart"],
+    },
+    {
+      name: "consul",
+      port: 8500,
+      exec: ["/bin/consul", "agent", "-server",
+             "-bootstrap-expect", "3",
```

Comment:
The problem is that this moves the `-bootstrap-expect 3` from the `docker-compose.yml` (where it is easily configured) into a place where it isn't.

That feature (being able to switch from demo-friendly non-HA to production-friendly HA) is being used in other blueprints.